### PR TITLE
add running scripts & FAS

### PR DESCRIPTION
**Pull request by ryestew**
fixes ethereum/remix-project/issues/414

### CODE REVIEW

**Comment by ryestew** [comment]
needs links to ethers & web3
fix the nav in index.rst
a few more scripts in FAS
link to the medium post about async await
settings doc needs an update.

**Comment by ryestew** [comment]
Still needs an update to the settings page (although this could go in another PR).  Maybe needs more scripts in the FAS page.

**Comment by ryestew** [comment]
Settings update should go into another PR.

**Review by Aniket-Engg — APPROVED** [review]
I think it is good to start with

File: docs/running_js_scripts.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Why run JavaScript Scripts in Remix?
+* To mimic how the front-end of your dapp will use web3.js or ethers.js
+* To quickly deploy and interact with a bunch of instances of a contract without going through the Remix GUI.
+* To run some tests on a previous deployed contract.
+
+## Setup
+1. Make sure the **Generate contract metadata** option is **checked** in the settings module.
+[Settings documentation](settings.html)  ( ROB TODO: and make sure to update Settings.md!!!)
+
+2. **Async/await scripts do not currently work on the JSVM**
```
Comment (yann300) [inline docs/running_js_scripts.md:15]: this will be fixed for the current release

File: docs/running_js_scripts.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+[Settings documentation](settings.html)  ( ROB TODO: and make sure to update Settings.md!!!)
+
+2. **Async/await scripts do not currently work on the JSVM**  
+    * Go to the Deploy and Run module to select Injected Web3 or Web3 Provider and NOT the JSVM. 
+
+3. Compile a solidity file.
+
+4. Write the script
+
+5. To run the script - either have the script be the active file in the editor and run `remix.exeCurrent()` in the console or right click on the script in the files explorer to get the popup context menu and select the **run** option. (ROB TODO - update with an image of the context menu)
```
Comment (yann300) [inline docs/running_js_scripts.md:22]: there's a TODO missing
`(ROB TODO - update with an image of the context menu)`

File: docs/running_js_scripts.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## An Example
+The example below is to deploy a solidity contract named CustomERC20.sol.  That contract had been compiled and its metadata generated and put into the browser/artifacts folder. This example is using the web3.js library.  Ethers.js could also be used.
+
+```
+(async () => {
```
Comment (yann300) [inline docs/running_js_scripts.md:28]: should this also be in FAS.md

Comment (ryestew) [reply docs/running_js_scripts.md:28]: It is in there.

**Pull request merged**